src/index.ts

```diff
@@ -22,8 +22,11 @@ interface TelescopeNode {
   telescopicOut: TelescopicOutput;
 }
 
+// time; recorded to prevent recursive text expansion on single hover
+let _lastHoveredTime = Date.now();
```

Comment:
nit: prefer `const` for unchanging constants

Reply:
Oh, it is used! See line 64

Reply:
oh nvm i see

Reply:
oh i get what you're doing cool